flow/flow.py

```diff
@@ -505,6 +505,15 @@ def handle_message(self, type, params):
                         json.dumps({    'controller_path': c.path_on_server(),
                                         'recording': False,
                                         'recording_interval': self.recording_interval }))
+            #
+            # Stop recording if in progress.
+            # Remove the currently running diagram program.
+            # Remove the currently set user.
+            #
+            self.recording_interval = None
+            self.recording_location = None
+            self.diagram            = None
+            self.username           = None
 
             if self.recording_location != stop_location:
```

Comment:
With the latest changes, self.recording_location will always be None at this point

Reply:
Yup, meant for the `recording_location` to be set to none after the comparison.  Thanks for catching!